docs/data-ai/capture-data/advanced/how-sync-works.md

```diff
@@ -0,0 +1,100 @@
+---
+linkTitle: "How sync works"
+title: "How sync works"
+tags: ["data management", "data", "services"]
+weight: 12
+layout: "docs"
+type: "docs"
+platformarea: ["data"]
+description: "TODO"
+date: "2024-12-18"
+---
+
+Data capture and cloud sync works differently for `viam-server` and `viam-micro-server`.
+
+{{< tabs >}}
+{{% tab name="viam-server" %}}
+
+The data is captured locally on the machine's storage and, by default, stored in the `~/.viam/capture` directory.
+
+If a machine restarts for any reason, capture automatically resumes and any data from already stored but not yet synced is synced.
+
+The service can capture data from multiple resources at the same or different frequencies.
+The service does not impose a lower or upper limit on the frequency of data collection.
+However, in practice, your hardware may impose limits on the frequency of data collection.
+Avoid configuring data capture to higher rates than your hardware can handle, as this could lead to performance degradation.
+
+Data capture is frequently used with cloud sync.
+However, if you want to manage your machine's captured data yourself, you can enable only data capture without cloud sync.
+
+{{% /tab %}}
+{{% tab name="viam-micro-server" %}}
+
+The data is captured in the ESP32's flash memory until it is uploaded to the Viam Cloud.
+
+If the machine restarts before all data is synced, all unsynced data captured since the last sync point is lost.
+
+The service can capture data from multiple resources at the same or different frequencies.
+The service does not impose a lower or upper limit on the frequency of data collection.
+However, in practice, high frequency data collection (> 100Hz) requires special considerations on the ESP32.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Security
+
+The data management service uses {{< glossary_tooltip term_id="grpc" text="gRPC" >}} calls to send and receive data, so your data is encrypted while in flight.
+When data is stored in the cloud, it is encrypted at rest by the cloud storage provider.
+
+## Data Integrity
+
+Viam's data management service is designed to safeguard against data loss, data duplication and otherwise compromised data.
+
+If the internet becomes unavailable or the machine needs to restart during the sync process, the sync is interrupted.
+If the sync process is interrupted, the service will retry uploading the data at exponentially increasing intervals until the interval in between tries is at one hour, at which point the service retries the sync every hour.
+When the connection is restored and sync resumes, the service continues sync where it left off without duplicating data.
+If the interruption happens mid-file, sync resumes from the beginning of that file.
+
+To avoid syncing files that are still being written to, the data management service only syncs files that haven't been modified in the previous 10 seconds.
+
+## Storage
+
```

Comment:
does seem to fit into how sync works but I think is also important enough to be in regular reference docs-- thoughts?

Reply:
I suppose this is also a reference page for it. I think the data page in reference right now is not visible, if that stays you can cover the info there too and see how that feels.

The automatic data deletion may need to go somewhere else instead I think, that goes beyond explaining how sync works